Tests iOS/Support/LaunchArguments.swift

```diff
@@ -54,6 +54,10 @@ struct LaunchArguments {
         if disableSnowplow {
             args.append("disableSnowplow")
         }
+
+        // Skip deleting the core data store in PersistentContainer by setting user defaults key to true.
+        args.append("-PersistentContainer.reset.data.02.21.2023 true")
```

Comment:
https://sarunw.com/posts/how-to-set-userdefaults-value-with-launch-arguments/

Reply:
thanks for linking resources in your PR 🔥 helpful to know the context :)